pallets/drand/Cargo.toml

```diff
@@ -86,3 +86,5 @@ try-runtime = [
 	"frame-system/try-runtime",
 	"sp-runtime/try-runtime",
 ]
+# use the drand mainnet configuration
+mainnet = []
```

Comment:
As per a comment below, I don't think we should specify the net as a feature, but as a pallet config
```suggestion
```